BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -0,0 +1,596 @@
+version 1.0
+
+
+# Object holding all reference files, to ensure all localized in tasks
+struct Reference {
+    File fasta
+    File index
+    File dict
+}
+
+# Object representing expression for subsetting VCFs
+struct VariantSelector {
+    String bcf_label
+    String bcf_selector
+}
+
+# Object representing file with intervals to subset analysis over
+struct Stratifier {
+    String label
+    File? interval
+}
+
+# Main workflow: performs evaluation of call_vcf against base_vcf using vcfeval comparison engine,
+# over different analysis modes.
+workflow SimpleBenchmark {
+    input {
+        # VCF information
+        File base_vcf
+        File base_vcf_index
+        String base_output_sample_name
+        String? base_vcf_sample_name
+
+        File call_vcf
+        File call_vcf_index
+        String call_output_sample_name
+        String? call_vcf_sample_name
+
+        # Reference information
+        File ref_fasta
+        File ref_index
+        File ref_dict
+
+        # Subsetting inputs using intervals
+        Array[File] strat_intervals = []
+        Array[String] strat_labels = []
+        Int interval_padding = 0
+        String subset_gatk_tag = "4.2.3.0"
+
+        # Subsetting inputs using variant properties; Note SNP & INDEL are separated automatically later
+        Array[String] bcf_selectors = []
+        Array[String] bcf_labels = []
+
+        # Evaluation inputs
+        File? evaluation_bed
```

Comment:
Yes, but also acts as a general purpose "global background" region all analysis will occur on. For example, this could be a "WGS bed" file in the absence of a truth bed.